src/ROUTES.js

```diff
@@ -187,4 +187,5 @@ export default {
             isSubReportPageRoute: pathSegments.length > 2,
         };
     },
+    SIGNINMODAL: 'signinmodal',
```

Comment:
I think it's better to separate the words with the dash and underscore for the key.

```suggestion
    SIGN_IN_MODAL: 'sign-in-modal',
```

Reply:
Updated name and references!